LinearApproach/LinearApproachMultiCCA.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>If you get the chance, please either change variable names to match Witten 2009 or specify your notation in a comment (which will be included in the docstring).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/sparsecca/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>changed PC to F, and samples to S, is that it?</p>

Reply:
<p>and model.Idx -&gt; model.N (N is number of matrices)</p>